lte/protos/session_manager.proto

```diff
@@ -567,3 +567,148 @@ service CentralSessionController {
   // Terminates session in OCS/PCRF for a subscriber
   rpc TerminateSession(SessionTerminateRequest) returns (SessionTerminateResponse) {}
 }
+
+
+
+/////////////////////////////////////////////////////////////////////////////////////////
+// SUMMARY of messages from PIPELINED (UPF) to SESSIOND (SMF)
+//   UPFNodeState                 -  Event Triggered
+//        UPFNodeReport           :  Contains any GTPU Peer Failure information
+//        UPFAssociationState     :  UPF Association state triggered during start up.
+//
+/////////////////////////////////////////////////////////////////////////////////////////
+
+
+message NodeID {
+    enum NodeIDType {
+         NONE = 0;
+         IPv4 = 1;
+         IPv6 = 2;
+         FQDN = 3;
+    }
+
+    NodeIDType idtype = 1;
+    string identifier = 2;
+}
+
+
+// Node Level Messages from UPF. Purely Event Based.
+message UPFNodeState {
+
+    oneof upf_node_messages {
+         // Message for establishing association
+         UPFAssociationState associaton_state = 2;
+
+         // Contains node information and any GTPU Peer failure report.
+         UPFNodeReport node_report = 1;
+
+    }
+}
+
+//15.8 (Sec 8.2.82 User Plane IP Resource Information)
+message UserPlaneIPResourceSchema {
+    string ipv4_address = 1;
+    string ipv6_address = 2;
+    uint32 teid_range_indication = 3;
+    uint32 teid_range = 4;
+    string assoc_network_instance = 5;
+
+    enum InterfaceValue {    // 15.8 (Sec 8.2.2) Source Interface
+       ACCESS = 0;
+       CORE = 1;
+       LAN  = 2;
+       CP_FUNCTION = 3;
+    }
+    uint32 assoc_source_interface = 6;
+}
+
+
+// 15.8 (Sec 7.4.4.3) : PFCP Association Update Reqest.
+message UPFAssociationState {
+  NodeID node = 1;                 // Mandatory and if this is only present indicates deletion
+
+  // Version of the associated state
+  sint32 state_version = 2;
```

Comment:
is there a reason why this is `sint32`?

Reply:
It was carried over from initial discussion. Have corrected and made it unsigned int.